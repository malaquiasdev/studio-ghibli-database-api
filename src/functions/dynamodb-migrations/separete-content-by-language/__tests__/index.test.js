const myNeighborTotoroRaw = require('../../../../../database/movies/my-neighbor-totoro.json');
const totoroWithoutDescriptionJALanguageMock = require('./__mocks__/totoro-without-description-ja-language-mock.json');
const separateContentByLanguage = require('../index');

describe('# Functions - Dynamodb Migrations - separateContentByLanguage', () => {
  it('should be a function', () => {
    expect(typeof separateContentByLanguage).toBe('function');
  });

  it('should return an array as result', () => {
    const emptyArray = separateContentByLanguage();
    expect(Array.isArray(emptyArray)).toBe(true);

    const array = separateContentByLanguage(myNeighborTotoroRaw);
    expect(Array.isArray(array)).toBe(true);
  });

  it('should return all contents that we have inside of the raw file as an array result', () => {
    const array = separateContentByLanguage([myNeighborTotoroRaw]);
    expect(array.length).toBe(3);
    expect(array).toMatchSnapshot();
  });

  it('should return a content with language as empty string because we do not have translation information', () => {
    const array = separateContentByLanguage([
      totoroWithoutDescriptionJALanguageMock,
    ]);
    const totoroJapaneseVersion = array.find(
      (item) => item.title === 'となりのトトロ',
    );
    expect(totoroJapaneseVersion.description).toBe('');
  });
});
