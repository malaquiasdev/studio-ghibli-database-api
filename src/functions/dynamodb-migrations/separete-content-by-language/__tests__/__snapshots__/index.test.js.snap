// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Functions - Dynamodb Migrations - separateContentByLanguage should return all contents that we have inside of the raw file as an array result 1`] = `
Array [
  Object {
    "budget": 3700000,
    "cast": Array [
      Object {
        "characterName": "Satsuki",
        "name": "Noriko Hidaka",
        "order": 0,
        "role": "Voice",
      },
      Object {
        "characterName": "Hitoshi Takagi",
        "name": "Totoro",
        "order": 1,
        "role": "Voice",
      },
      Object {
        "characterName": "Mei",
        "name": "Chika Sakamoto",
        "order": 2,
        "role": "Voice",
      },
      Object {
        "characterName": "Tatsuo Kusakabe",
        "name": "Shigesato Itoi",
        "order": 3,
        "role": "Voice",
      },
      Object {
        "characterName": "Yasuko Kusakabe",
        "name": "Sumi Shimamoto",
        "order": 4,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no obâsan",
        "name": "Tanie Kitabayashi",
        "order": 5,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no otôsan",
        "name": "Masashi Hirose",
        "order": 6,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no okâsan",
        "name": "Yûko Maruyama",
        "order": 7,
        "role": "Voice",
      },
      Object {
        "characterName": "Kusakari-Otoko",
        "name": "Shigeru Chiba",
        "order": 8,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta",
        "name": "Toshiyuki Amagasa",
        "order": 9,
        "role": "Voice",
      },
    ],
    "crew": Array [
      Object {
        "name": "Tensai Okamura",
        "order": 0,
        "role": "Animation",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 1,
        "role": "Director",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 2,
        "role": "Writer",
      },
      Object {
        "name": "Rick Dempsey",
        "order": 3,
        "role": "Executive Producer",
      },
      Object {
        "name": "Mark Henley",
        "order": 4,
        "role": "Director of Photography",
      },
      Object {
        "name": "Ned Lott",
        "order": 5,
        "role": "Producer",
      },
      Object {
        "name": "Ned Lott",
        "order": 6,
        "role": "Casting",
      },
      Object {
        "name": "Joe Hisaishi",
        "order": 7,
        "role": "Original Music Composer",
      },
      Object {
        "name": "Toshio Suzuki",
        "order": 8,
        "role": "Producer",
      },
      Object {
        "name": "Takeshi Seyama",
        "order": 9,
        "role": "Editor",
      },
      Object {
        "name": "Yasuyoshi Tokuma",
        "order": 10,
        "role": "Executive Producer",
      },
      Object {
        "name": "Yōji Takeshige",
        "order": 11,
        "role": "Background Designer",
      },
      Object {
        "name": "Makiko Futaki",
        "order": 12,
        "role": "Animation",
      },
      Object {
        "name": "Hisao Shirai",
        "order": 13,
        "role": "Director of Photography",
      },
      Object {
        "name": "Tooru Hara",
        "order": 14,
        "role": "Producer",
      },
      Object {
        "name": "Shinji Kimura",
        "order": 15,
        "role": "Background Designer",
      },
      Object {
        "name": "Katsuya Kondo",
        "order": 16,
        "role": "Animation",
      },
      Object {
        "name": "Eiko Tanaka",
        "order": 17,
        "role": "Line Producer",
      },
      Object {
        "name": "Sunao Katabuchi",
        "order": 18,
        "role": "Background Designer",
      },
      Object {
        "name": "Kazuo Oga",
        "order": 19,
        "role": "Art Direction",
      },
      Object {
        "name": "Toshio Nozaki",
        "order": 20,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshinori Kanada",
        "order": 21,
        "role": "Animation",
      },
      Object {
        "name": "Hidetoshi Kaneko",
        "order": 22,
        "role": "Background Designer",
      },
      Object {
        "name": "Masaaki Endou",
        "order": 23,
        "role": "Animation",
      },
      Object {
        "name": "Shinji Otsuka",
        "order": 24,
        "role": "Animation",
      },
      Object {
        "name": "Hirômi Yamakawa",
        "order": 25,
        "role": "Animation",
      },
      Object {
        "name": "Toshio Kawaguchi",
        "order": 26,
        "role": "Animation",
      },
      Object {
        "name": "Masako Shinohara",
        "order": 27,
        "role": "Animation",
      },
      Object {
        "name": "Hitomi Tateno",
        "order": 28,
        "role": "Animation Manager",
      },
      Object {
        "name": "Michiyo Yasuda",
        "order": 29,
        "role": "Painter",
      },
      Object {
        "name": "Kiyomi Ôta",
        "order": 30,
        "role": "Background Designer",
      },
      Object {
        "name": "Junko Ina",
        "order": 31,
        "role": "Background Designer",
      },
      Object {
        "name": "Komasa",
        "order": 32,
        "role": "Animation",
      },
      Object {
        "name": "Tsuyoshi Matsumuro",
        "order": 33,
        "role": "Background Designer",
      },
      Object {
        "name": "Hajime Matsuoka",
        "order": 34,
        "role": "Background Designer",
      },
      Object {
        "name": "Yuko Matsuura",
        "order": 35,
        "role": "Background Designer",
      },
      Object {
        "name": "Nobuhiro Otsuka",
        "order": 36,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshiharu Sato",
        "order": 37,
        "role": "Animation Supervisor",
      },
      Object {
        "name": "Makoto Shiraishi",
        "order": 38,
        "role": "Background Designer",
      },
      Object {
        "name": "Kiyoko Sugawara",
        "order": 39,
        "role": "Background Designer",
      },
      Object {
        "name": "Yasuko Tachiki",
        "order": 40,
        "role": "Animation Manager",
      },
      Object {
        "name": "Hideko Tagawa",
        "order": 41,
        "role": "Animation",
      },
      Object {
        "name": "Keiko Tamura",
        "order": 42,
        "role": "Background Designer",
      },
      Object {
        "name": "Makoto Tanaka",
        "order": 43,
        "role": "Animation",
      },
      Object {
        "name": "Sadahiko Tanaka",
        "order": 44,
        "role": "Background Designer",
      },
      Object {
        "name": "Tsukasa Tannai",
        "order": 45,
        "role": "Animation",
      },
      Object {
        "name": "Masaki Yoshizaki",
        "order": 46,
        "role": "Background Designer",
      },
      Object {
        "name": "Tetsuya Endō",
        "order": 47,
        "role": "Assistant Director",
      },
      Object {
        "name": "Akira Yamakawa",
        "order": 48,
        "role": "Background Designer",
      },
    ],
    "description": "都会からそんなに遠くない田舎町に引っ越してきたサツキとメイ。お化けやしきみたいに古い家には、何かが棲んでいる気配がいっぱい。サツキたちの家の裏にある神社の大木をすみかにしているトトロ。人の住んでいない家をススやホコリだらけにしてしまうススワタリ（まっくろくろすけ）。大きな目と大きな口、そして何本もある足で空間を自由自在に走りまわる巨大なネコバス。サツキとメイは次々に奇妙ないきものと出会い、ステキな冒険に飛び出す。",
    "genres": Array [
      "アニメーション",
      "ファミリー",
      "ファンタジー",
    ],
    "id": "MV03222",
    "language": "ja",
    "originalLanguage": Object {
      "iso6391": "ja",
      "name": "Japanese",
    },
    "ratings": Array [
      Object {
        "key": "Departamento de Justiça, Classificação, Títulos e Qualificação",
        "value": "L",
      },
      Object {
        "key": "Motion Picture Association of America",
        "value": "G",
      },
    ],
    "releaseYear": "1988",
    "revenue": 45000000,
    "runTime": 86,
    "title": "となりのトトロ",
    "type": "movie",
  },
  Object {
    "budget": 3700000,
    "cast": Array [
      Object {
        "characterName": "Satsuki",
        "name": "Noriko Hidaka",
        "order": 0,
        "role": "Voice",
      },
      Object {
        "characterName": "Hitoshi Takagi",
        "name": "Totoro",
        "order": 1,
        "role": "Voice",
      },
      Object {
        "characterName": "Mei",
        "name": "Chika Sakamoto",
        "order": 2,
        "role": "Voice",
      },
      Object {
        "characterName": "Tatsuo Kusakabe",
        "name": "Shigesato Itoi",
        "order": 3,
        "role": "Voice",
      },
      Object {
        "characterName": "Yasuko Kusakabe",
        "name": "Sumi Shimamoto",
        "order": 4,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no obâsan",
        "name": "Tanie Kitabayashi",
        "order": 5,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no otôsan",
        "name": "Masashi Hirose",
        "order": 6,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no okâsan",
        "name": "Yûko Maruyama",
        "order": 7,
        "role": "Voice",
      },
      Object {
        "characterName": "Kusakari-Otoko",
        "name": "Shigeru Chiba",
        "order": 8,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta",
        "name": "Toshiyuki Amagasa",
        "order": 9,
        "role": "Voice",
      },
    ],
    "crew": Array [
      Object {
        "name": "Tensai Okamura",
        "order": 0,
        "role": "Animation",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 1,
        "role": "Director",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 2,
        "role": "Writer",
      },
      Object {
        "name": "Rick Dempsey",
        "order": 3,
        "role": "Executive Producer",
      },
      Object {
        "name": "Mark Henley",
        "order": 4,
        "role": "Director of Photography",
      },
      Object {
        "name": "Ned Lott",
        "order": 5,
        "role": "Producer",
      },
      Object {
        "name": "Ned Lott",
        "order": 6,
        "role": "Casting",
      },
      Object {
        "name": "Joe Hisaishi",
        "order": 7,
        "role": "Original Music Composer",
      },
      Object {
        "name": "Toshio Suzuki",
        "order": 8,
        "role": "Producer",
      },
      Object {
        "name": "Takeshi Seyama",
        "order": 9,
        "role": "Editor",
      },
      Object {
        "name": "Yasuyoshi Tokuma",
        "order": 10,
        "role": "Executive Producer",
      },
      Object {
        "name": "Yōji Takeshige",
        "order": 11,
        "role": "Background Designer",
      },
      Object {
        "name": "Makiko Futaki",
        "order": 12,
        "role": "Animation",
      },
      Object {
        "name": "Hisao Shirai",
        "order": 13,
        "role": "Director of Photography",
      },
      Object {
        "name": "Tooru Hara",
        "order": 14,
        "role": "Producer",
      },
      Object {
        "name": "Shinji Kimura",
        "order": 15,
        "role": "Background Designer",
      },
      Object {
        "name": "Katsuya Kondo",
        "order": 16,
        "role": "Animation",
      },
      Object {
        "name": "Eiko Tanaka",
        "order": 17,
        "role": "Line Producer",
      },
      Object {
        "name": "Sunao Katabuchi",
        "order": 18,
        "role": "Background Designer",
      },
      Object {
        "name": "Kazuo Oga",
        "order": 19,
        "role": "Art Direction",
      },
      Object {
        "name": "Toshio Nozaki",
        "order": 20,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshinori Kanada",
        "order": 21,
        "role": "Animation",
      },
      Object {
        "name": "Hidetoshi Kaneko",
        "order": 22,
        "role": "Background Designer",
      },
      Object {
        "name": "Masaaki Endou",
        "order": 23,
        "role": "Animation",
      },
      Object {
        "name": "Shinji Otsuka",
        "order": 24,
        "role": "Animation",
      },
      Object {
        "name": "Hirômi Yamakawa",
        "order": 25,
        "role": "Animation",
      },
      Object {
        "name": "Toshio Kawaguchi",
        "order": 26,
        "role": "Animation",
      },
      Object {
        "name": "Masako Shinohara",
        "order": 27,
        "role": "Animation",
      },
      Object {
        "name": "Hitomi Tateno",
        "order": 28,
        "role": "Animation Manager",
      },
      Object {
        "name": "Michiyo Yasuda",
        "order": 29,
        "role": "Painter",
      },
      Object {
        "name": "Kiyomi Ôta",
        "order": 30,
        "role": "Background Designer",
      },
      Object {
        "name": "Junko Ina",
        "order": 31,
        "role": "Background Designer",
      },
      Object {
        "name": "Komasa",
        "order": 32,
        "role": "Animation",
      },
      Object {
        "name": "Tsuyoshi Matsumuro",
        "order": 33,
        "role": "Background Designer",
      },
      Object {
        "name": "Hajime Matsuoka",
        "order": 34,
        "role": "Background Designer",
      },
      Object {
        "name": "Yuko Matsuura",
        "order": 35,
        "role": "Background Designer",
      },
      Object {
        "name": "Nobuhiro Otsuka",
        "order": 36,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshiharu Sato",
        "order": 37,
        "role": "Animation Supervisor",
      },
      Object {
        "name": "Makoto Shiraishi",
        "order": 38,
        "role": "Background Designer",
      },
      Object {
        "name": "Kiyoko Sugawara",
        "order": 39,
        "role": "Background Designer",
      },
      Object {
        "name": "Yasuko Tachiki",
        "order": 40,
        "role": "Animation Manager",
      },
      Object {
        "name": "Hideko Tagawa",
        "order": 41,
        "role": "Animation",
      },
      Object {
        "name": "Keiko Tamura",
        "order": 42,
        "role": "Background Designer",
      },
      Object {
        "name": "Makoto Tanaka",
        "order": 43,
        "role": "Animation",
      },
      Object {
        "name": "Sadahiko Tanaka",
        "order": 44,
        "role": "Background Designer",
      },
      Object {
        "name": "Tsukasa Tannai",
        "order": 45,
        "role": "Animation",
      },
      Object {
        "name": "Masaki Yoshizaki",
        "order": 46,
        "role": "Background Designer",
      },
      Object {
        "name": "Tetsuya Endō",
        "order": 47,
        "role": "Assistant Director",
      },
      Object {
        "name": "Akira Yamakawa",
        "order": 48,
        "role": "Background Designer",
      },
    ],
    "description": "Two sisters move to the country with their father in order to be closer to their hospitalized mother, and discover the surrounding trees are inhabited by Totoros, magical spirits of the forest. When the youngest runs away from home, the older sister seeks help from the spirits to find her.",
    "genres": Array [
      "Animation",
      "Family",
      "Fantasy",
    ],
    "id": "MV03222",
    "language": "en-US",
    "originalLanguage": Object {
      "iso6391": "ja",
      "name": "Japanese",
    },
    "ratings": Array [
      Object {
        "key": "Departamento de Justiça, Classificação, Títulos e Qualificação",
        "value": "L",
      },
      Object {
        "key": "Motion Picture Association of America",
        "value": "G",
      },
    ],
    "releaseYear": "1988",
    "revenue": 45000000,
    "runTime": 86,
    "title": "My Neighbor Totoro",
    "type": "movie",
  },
  Object {
    "budget": 3700000,
    "cast": Array [
      Object {
        "characterName": "Satsuki",
        "name": "Noriko Hidaka",
        "order": 0,
        "role": "Voice",
      },
      Object {
        "characterName": "Hitoshi Takagi",
        "name": "Totoro",
        "order": 1,
        "role": "Voice",
      },
      Object {
        "characterName": "Mei",
        "name": "Chika Sakamoto",
        "order": 2,
        "role": "Voice",
      },
      Object {
        "characterName": "Tatsuo Kusakabe",
        "name": "Shigesato Itoi",
        "order": 3,
        "role": "Voice",
      },
      Object {
        "characterName": "Yasuko Kusakabe",
        "name": "Sumi Shimamoto",
        "order": 4,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no obâsan",
        "name": "Tanie Kitabayashi",
        "order": 5,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no otôsan",
        "name": "Masashi Hirose",
        "order": 6,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta no okâsan",
        "name": "Yûko Maruyama",
        "order": 7,
        "role": "Voice",
      },
      Object {
        "characterName": "Kusakari-Otoko",
        "name": "Shigeru Chiba",
        "order": 8,
        "role": "Voice",
      },
      Object {
        "characterName": "Kanta",
        "name": "Toshiyuki Amagasa",
        "order": 9,
        "role": "Voice",
      },
    ],
    "crew": Array [
      Object {
        "name": "Tensai Okamura",
        "order": 0,
        "role": "Animation",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 1,
        "role": "Director",
      },
      Object {
        "name": "Hayao Miyazaki",
        "order": 2,
        "role": "Writer",
      },
      Object {
        "name": "Rick Dempsey",
        "order": 3,
        "role": "Executive Producer",
      },
      Object {
        "name": "Mark Henley",
        "order": 4,
        "role": "Director of Photography",
      },
      Object {
        "name": "Ned Lott",
        "order": 5,
        "role": "Producer",
      },
      Object {
        "name": "Ned Lott",
        "order": 6,
        "role": "Casting",
      },
      Object {
        "name": "Joe Hisaishi",
        "order": 7,
        "role": "Original Music Composer",
      },
      Object {
        "name": "Toshio Suzuki",
        "order": 8,
        "role": "Producer",
      },
      Object {
        "name": "Takeshi Seyama",
        "order": 9,
        "role": "Editor",
      },
      Object {
        "name": "Yasuyoshi Tokuma",
        "order": 10,
        "role": "Executive Producer",
      },
      Object {
        "name": "Yōji Takeshige",
        "order": 11,
        "role": "Background Designer",
      },
      Object {
        "name": "Makiko Futaki",
        "order": 12,
        "role": "Animation",
      },
      Object {
        "name": "Hisao Shirai",
        "order": 13,
        "role": "Director of Photography",
      },
      Object {
        "name": "Tooru Hara",
        "order": 14,
        "role": "Producer",
      },
      Object {
        "name": "Shinji Kimura",
        "order": 15,
        "role": "Background Designer",
      },
      Object {
        "name": "Katsuya Kondo",
        "order": 16,
        "role": "Animation",
      },
      Object {
        "name": "Eiko Tanaka",
        "order": 17,
        "role": "Line Producer",
      },
      Object {
        "name": "Sunao Katabuchi",
        "order": 18,
        "role": "Background Designer",
      },
      Object {
        "name": "Kazuo Oga",
        "order": 19,
        "role": "Art Direction",
      },
      Object {
        "name": "Toshio Nozaki",
        "order": 20,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshinori Kanada",
        "order": 21,
        "role": "Animation",
      },
      Object {
        "name": "Hidetoshi Kaneko",
        "order": 22,
        "role": "Background Designer",
      },
      Object {
        "name": "Masaaki Endou",
        "order": 23,
        "role": "Animation",
      },
      Object {
        "name": "Shinji Otsuka",
        "order": 24,
        "role": "Animation",
      },
      Object {
        "name": "Hirômi Yamakawa",
        "order": 25,
        "role": "Animation",
      },
      Object {
        "name": "Toshio Kawaguchi",
        "order": 26,
        "role": "Animation",
      },
      Object {
        "name": "Masako Shinohara",
        "order": 27,
        "role": "Animation",
      },
      Object {
        "name": "Hitomi Tateno",
        "order": 28,
        "role": "Animation Manager",
      },
      Object {
        "name": "Michiyo Yasuda",
        "order": 29,
        "role": "Painter",
      },
      Object {
        "name": "Kiyomi Ôta",
        "order": 30,
        "role": "Background Designer",
      },
      Object {
        "name": "Junko Ina",
        "order": 31,
        "role": "Background Designer",
      },
      Object {
        "name": "Komasa",
        "order": 32,
        "role": "Animation",
      },
      Object {
        "name": "Tsuyoshi Matsumuro",
        "order": 33,
        "role": "Background Designer",
      },
      Object {
        "name": "Hajime Matsuoka",
        "order": 34,
        "role": "Background Designer",
      },
      Object {
        "name": "Yuko Matsuura",
        "order": 35,
        "role": "Background Designer",
      },
      Object {
        "name": "Nobuhiro Otsuka",
        "order": 36,
        "role": "Background Designer",
      },
      Object {
        "name": "Yoshiharu Sato",
        "order": 37,
        "role": "Animation Supervisor",
      },
      Object {
        "name": "Makoto Shiraishi",
        "order": 38,
        "role": "Background Designer",
      },
      Object {
        "name": "Kiyoko Sugawara",
        "order": 39,
        "role": "Background Designer",
      },
      Object {
        "name": "Yasuko Tachiki",
        "order": 40,
        "role": "Animation Manager",
      },
      Object {
        "name": "Hideko Tagawa",
        "order": 41,
        "role": "Animation",
      },
      Object {
        "name": "Keiko Tamura",
        "order": 42,
        "role": "Background Designer",
      },
      Object {
        "name": "Makoto Tanaka",
        "order": 43,
        "role": "Animation",
      },
      Object {
        "name": "Sadahiko Tanaka",
        "order": 44,
        "role": "Background Designer",
      },
      Object {
        "name": "Tsukasa Tannai",
        "order": 45,
        "role": "Animation",
      },
      Object {
        "name": "Masaki Yoshizaki",
        "order": 46,
        "role": "Background Designer",
      },
      Object {
        "name": "Tetsuya Endō",
        "order": 47,
        "role": "Assistant Director",
      },
      Object {
        "name": "Akira Yamakawa",
        "order": 48,
        "role": "Background Designer",
      },
    ],
    "description": "Mei é uma jovem que encontra uma pequena passagem em seu quintal, que a leva à um lendário espírito da floresta conhecido como Totoro. Sua mãe está no hospital e seu pai divide o tempo entre dar aulas na faculdade e cuidar de sua mulher doente. Quando Mei tenta visitar a mãe por conta própria, se perde na floresta e só o grande e fofo Totoro pode ajudar a menina a achar o caminho de volta para casa.",
    "genres": Array [
      "Animação",
      "Família",
      "Fantasia",
    ],
    "id": "MV03222",
    "language": "pt-BR",
    "originalLanguage": Object {
      "iso6391": "ja",
      "name": "Japanese",
    },
    "ratings": Array [
      Object {
        "key": "Departamento de Justiça, Classificação, Títulos e Qualificação",
        "value": "L",
      },
      Object {
        "key": "Motion Picture Association of America",
        "value": "G",
      },
    ],
    "releaseYear": "1988",
    "revenue": 45000000,
    "runTime": 86,
    "title": "Meu Amigo Totoro",
    "type": "movie",
  },
]
`;
